.jules/bolt.md

```
@@ -0,0 +1,7 @@
+## 2026-02-11 - UseWindowDimensions Debounce
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)